### ISSUE TITLE
Python syntax highlighting 

### ISSUE THREAD

**Issue by sravanareddy** [sravanareddy/cs111grader#7]
As per Eni's suggestion, syntax-highlight test case descriptions. This can either be done in `produceOutput`, or directly in `testCaseReader.py`. 

http://pygments.org/docs/quickstart/ may help with this.

**Comment by sravanareddy** [comment]
(This or anything filed under the "enhancements" label is a nice-to-have, but not urgent, feature. Work on it only after the "todo" and "bug" tasks are done.)

**Comment by sravanareddy** [comment]
Now that I think about it, it's better to do this in `produceOutput` so that the test case files remain clean.

**Comment by dkreimer** [comment]
Good point

**Comment by sravanareddy** [comment]
Figure out how to escape regions of the comment from getting highlighted. For example, I'd like to enter 

`beats('rock','scissors') returns True \if gesture1 beats gesture2;\ False otherwise`

(with \ or some other escape symbol) and have the syntax highlighter ignore `if gesture1 beats gesture2;`
